### PR TITLE
Admob

### PR DESCRIPTION
**Pull request by bearparadigm**
Setting the right ids to the project for it to request ads in mobile. "There should be a loading screen, or a fade out that locks input when requesting an ad, until the request is over, otherwise the button repeats, delaying delivery further"

### CODE REVIEW

File: Assets/Code/Main/AdInstaller.cs [hunk 1]
```diff
@@ -50,7 +50,7 @@ public async Task InitInstaller()
             var rewardID = adPlacementDetails?.RewardId;
 
             GoogleAdmob = new GoogleAdmob(PlayfabAdConfiguration.APP_ID_AD,
-                 PlayfabAdConfiguration.NAME_ONE_VIDEO_THREE_HINTS_UNIT_ID_TEST, PlayfabAdConfiguration.ONE_VIDEO_THREE_HINTS_UNIT_ID_TEST);
+                 PlayfabAdConfiguration.NAME_ONE_VIDEO_THREE_HINTS_UNIT_ID_TEST, PlayfabAdConfiguration.ONE_VIDEO_THREE_HINTS_UNIT_ID);
```
Comment (NotBugGreg) [inline Assets/Code/Main/AdInstaller.cs]: can you create a method with the following logic? Select test ids when the app is being running on unity or Debug build. Select production ids when it is a Release build

**Review by NotBugGreg — CHANGES_REQUESTED** [review]
please check my request

**Comment by bearparadigm** [comment]
the request has been fulfilled, please accept my changes '3'

File: Assets/Code/Basic Implementation/PlayfabAdConfiguration.cs [hunk 2]
```diff
@@ -3,19 +3,14 @@
...
-
-        //public static readonly string APP_ID_AD = "ca-app-pub-3009865580436574~3542055920";
-        //public static readonly string ONE_VIDEO_THREE_HINTS_UNIT_ID = "ca-app-pub-3009865580436574/8781940162";
-        //public static readonly string ONE_VIDEO_THREE_HINTS_UNIT_ID_TEST = "ca-app-pub-3940256099942544/5224354917";
-        //
-        //public static readonly string NAME_ONE_VIDEO_THREE_HINTS_UNIT_ID = "ONE_VIDEO_THREE_HINTS_UNIT_ID";
-        //public static readonly string NAME_ONE_VIDEO_THREE_HINTS_UNIT_ID_TEST = "ONE_VIDEO_THREE_HINTS_UNIT_ID_TEST";
+#if UNITY_EDITOR
+        public static readonly string AD_ID = "AdmobDoesntMind";
+        public static readonly string REWARD_ID = "EDITOR_SOLITAIRE_HINT_REWARD"; //for playfab
```
Comment (NotBugGreg) [inline Assets/Code/Basic Implementation/PlayfabAdConfiguration.cs]: the values correct for test are -> 

- ONE_VIDEO_THREE_HINTS_UNIT_ID_TEST = "ca-app-pub-3940256099942544/5224354917"
- NAME_ONE_VIDEO_THREE_HINTS_UNIT_ID_TEST = "ONE_VIDEO_THREE_HINTS_UNIT_ID_TEST"

Comment (NotBugGreg) [reply Assets/Code/Basic Implementation/PlayfabAdConfiguration.cs]: and when I mentioned debug build I referred to this -> https://docs.unity3d.com/ScriptReference/Debug-isDebugBuild.html

Look for something similar for the release build

**Review by NotBugGreg — CHANGES_REQUESTED** [review]
check my comments

**Review by NotBugGreg — APPROVED** [review]

**Pull request merged**